etc/avocado/avocado.conf

```diff
@@ -83,3 +83,10 @@ skip_broken_plugin_notification = []
 # options or test types).
 # The keyword "@DEFAULT" will be replaced with all available unused loaders.
 loaders = ['file', '@DEFAULT']
+
+# Options that will only have effect when the
+# corresponding optional plugin is installed
+[optional.plugins]
+# Plugin: resultsdb
+# URL of the resultsdb API
+# resultsdb_api = http://resultsdb.example.com/api/v2.0/
```

Comment:
I was trying to avoid unecessary work from you (sending new versions with smaller changes) :)